heron/tools/cli/src/python/execute.py

```diff
@@ -116,8 +119,8 @@ def heron_pex(topology_pex, topology_class_name, args=None):
     cmd = [topology_pex]
     if args is not None:
       cmd.extend(args)
-    Log.debug('$> %s' % ' '.join(cmd))
-    Log.debug('Heron options: %s' % str(heron_env['HERON_OPTIONS']))
+    Log.debug('$> %s', ' '.join(cmd))
```

Comment:
above you refactored out the '$> ' log syntax for something more descriptive. we should do the same here too.